rollup.config.js

```diff
@@ -27,6 +27,7 @@ function createEntries() {
 function createEntry(config) {
   const isGlobalBuild = config.format === 'iife'
   const isBundlerBuild = config.format !== 'iife' && !config.browser
+  const isBundlerESMBuild = /esm-bundler/.test(format)
```

Comment:
I wonder if it should be `const isBundlerESMBuild = config.format === 'es'`
otherwise I cannot build and told me `format is not defined`

Reply:
Oh damn sorry, it should be `config.format` 😓  Fixed!

Reply:
But why it can pass unit test🤔, and I think isBundlerESMBuild will always be `false`, because no `config.format` can pass the regex

Reply:
Yeah because I think I'm drunk or something... it should be `config.file`... 🤮

Reply:
I think it's finally fixed...!

Reply:
.......I think `isBundlerESMBuild` is still aways be `false`, it may be `config.file.includes('esm-bundler')`. But I know what you mean, and I think it's better to be `const isBundlerESMBuild = config.format === 'es'`, which is consistent in format

Reply:
OK good point. Just updated to not include `esm-browser` builds.

```js
const isBundlerESMBuild = config.format === 'es' && !config.browser
```

Reply:
Yes! I think it's finished now.

Reply:
🙌 🙌 🙌 🙌 🙌 

Thank you so much for your patients! 🤝